flow-typed/npm/typescript_v3.3.x.js

```diff
@@ -3300,7 +3300,7 @@ declare module "typescript" {
     ...
```

Comment:
are these changes from the flow-typed repo?

Reply:
No(t yet), I was testing what changes are needed in the first place